test/unit/StakingProducts/setProducts.js

```diff
@@ -141,6 +142,41 @@ describe('setProducts unit tests', function () {
     verifyProduct(product2, { ...productEditParams[1], bumpedPriceUpdateTime: latestTimestamp });
   });
 
+  it('should edit targetPrice and update bumpedPrice and bumpedPriceUpdateTime', async function () {
+    const { stakingProducts, cover } = this;
+    const [manager] = this.accounts.members;
+
+    const products = [{ ...newProductTemplate }];
+
+    const { _initialPrices } = await cover.getPriceAndCapacityRatios(products.map(p => p.productId));
+
+    // set products
+    await stakingProducts.connect(manager).setProducts(poolId, products);
+
+    const { timestamp: initialTimestamp } = await ethers.provider.getBlock('latest');
+
+    const { bumpedPrice: bumpedPriceBefore, bumpedPriceUpdateTime: bumpedPriceUpdateTimeBefore } =
+      await stakingProducts.getProduct(poolId, newProductTemplate.productId);
+
+    expect(bumpedPriceBefore).to.be.equal(_initialPrices[0]);
+    expect(bumpedPriceUpdateTimeBefore).to.be.equal(initialTimestamp);
+
+    await increaseTime(daysToSeconds(2)); // 1% bump
+    const bumpPercentage = BigNumber.from(100); // 1%
+
+    // increase targetPrice
+    const productEditParams = [{ ...products[0], targetPrice: 2000 }];
```

Comment:
do we have an equivalent test case for a target price decrease?

That's what actually happened in prod